message_ix_models/project/ssp/script/util/functions.py

```diff
@@ -371,27 +371,27 @@ def add_ccs_setup(scen: message_ix.Scenario, ssp="SSP2"):
         ccs_ssp_pars = {
             "LED": {
                 "co2storage": 0.25,
-                "co2rate": 4000 / 3.667,
+                "co2rate": 6000 / 3.667,
```

Comment:
for future reference, these are the lines that define the ssp assumptions of injection limits and storage limits. see issue #267

Reply:
Noted! Thanks!